libglusterfs/src/common-utils.c

```diff
@@ -5216,12 +5217,24 @@ close_fds_except(int *fdv, size_t count)
             }
         }
         if (should_close)
-            sys_close(i);
+            closer(i, prm);
     }
 #endif /* !GF_LINUX_HOST_OS */
     return 0;
 }
 
+void
+closer_close(int fd, void *prm)
+{
+    sys_close(fd);
+}
```

Comment:
As this should not be exported (see other comment), suggest to make it also static.